pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [
     "setuptools>=54.0", 
     "setuptools_scm[toml]>=5.0,<7.0", 
     "wheel>=0.36.2", 
-    "Cython>=0.29.22"
+    "Cython==~0.29"
```

Comment:
```suggestion
    "Cython~=0.29"
```

Reply:
i already changed it to that suggested version - checking if pull request was not updated;
we need that really online.